### PR TITLE
update long word formatting - service card

### PR DESCRIPTION
**Pull request by mikkelcodes**

### CODE REVIEW

File: packages/website/src/components/servicesCard/index.js [hunk 1]
```diff
@@ -40,3 +22,24 @@ export const ServicesCard = ({
 };
 
 export default ServicesCard;
+
+const SplitWord = ({ word }) => {
```
Comment (TheKnarf) [inline packages/website/src/components/servicesCard/index.js:26]: Is this really needed? Isn't it enough to do `text-overflow: ellipsis` or something similare?

File: packages/website/src/components/servicesCard/index.js [hunk 2]
```diff
@@ -40,3 +22,24 @@ export const ServicesCard = ({
...
+  const longWord = 15;
+  let formattedWord;
+
+  if (word.includes('-') && word.length > longWord)
+    formattedWord = word.split('-');
+
+  return (
+    <>
+      {formattedWord &&
```
Comment (TheKnarf) [inline packages/website/src/components/servicesCard/index.js:35]: Just use `if` and have two return statements...

**Review by TheKnarf — CHANGES_REQUESTED** [review]

File: packages/website/src/components/servicesCard/index.js (hunk 1 above)
Comment (mikkelcodes) [reply packages/website/src/components/servicesCard/index.js:26]: I think it is :/ These are titles like "systemutviklingsledelse" that are split into:
"systemutviklings-
ledelse"
and ending it with "systemutviklings..." might not be practical. But the method you proposed could be applied to blog post card descriptions to simplify things.

Comment (TheKnarf) [reply packages/website/src/components/servicesCard/index.js:26]: Css hypens then? https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens

Comment (mikkelcodes) [reply packages/website/src/components/servicesCard/index.js:26]: Screenshot 2021-04-14 at 19 40 17

Is this an acceptable look? (css hyphens)

Comment (TheKnarf) [reply packages/website/src/components/servicesCard/index.js]: Seems fine for me.

File: packages/website/src/components/servicesCard/index.js [hunk 3]
```diff
@@ -40,3 +22,20 @@ export const ServicesCard = ({
 };
 
 export default ServicesCard;
+
+const SplitWord = ({ titleSplit, title }) => (
```
Comment (TheKnarf) [inline packages/website/src/components/servicesCard/index.js:26]: Remove unused component.

**Review by TheKnarf — CHANGES_REQUESTED** [review]

File: packages/website/src/components/navyIntro/index.js [hunk 4]
```diff
@@ -3,23 +3,16 @@ import BackgroundImage from 'gatsby-background-image-es5';
 import { Title } from '../title';
 import { Description } from '../description';
 
-export const NavyIntro = ({ title, description, image, headingSplit }) => (
+export const NavyIntro = ({ title, description, image }) => (
   <>
     {image ? (
       <div className="sm:z-40 -mt-35 -mb-2">
         <BackgroundImage fluid={image}>
-          <div className="bg-navy bg-opacity-75 w-full pt-10 sm:h-50vh h-screen flex flex-col justify-center items-center sm:pt-16 sm:pb-16 pb-4">
```
Comment (mikkelcodes) [inline packages/website/src/components/navyIntro/index.js:11]: This line is from and resolves issue #301 but the same file also contains fixes for #307 (wordwrap)

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**